xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -763,7 +763,7 @@ std::string CGUIWindowVideoBase::GetResumeString(const CFileItem &item)
 
 bool CGUIWindowVideoBase::ShowResumeMenu(CFileItem &item)
 {
-  if (!item.m_bIsFolder && !item.IsPVR())
+  if (!item.m_bIsFolder && !item.IsPVR() || item.IsPlugin())
```

Comment:
I am using:

dirItems is an array of (u, list_item, folder)
xbmcplugin.addDirectoryItems(pluginhandle, dirItems)

and folder is being set correctly, it is false for the items I am adding.

This is occurring when I hit the play button on the media info screen. It does not happen when I select play for the item in the dir list, but if I hit Info to show info screen and then hit the play this is when I see item.m_bIsFolder being True